src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -184,15 +194,53 @@ const orderedSlides = computed(() => {
 		showedSlides = achievementSlides;
 	}
 
-	const sortedSlides = showedSlides.sort((a, b) => {
+	let sortedSlides = showedSlides.sort((a, b) => {
 		return a.milestoneDiff - b.milestoneDiff;
 	});
 
+	const nonBadgesSlides = props.slides.filter(slide => {
+		const richTextUiSettingsData = getRichTextUiSettingsData(slide);
+		return !defaultBadges.includes(richTextUiSettingsData.achievementKey);
+	});
+
+	if (nonBadgesSlides.length > 0) {
+		sortedSlides = [
+			...sortedSlides,
+			...nonBadgesSlides,
+		];
+	}
+
 	return sortedSlides;
 });
 
+const isNonBadgeSlide = slide => {
+	const richTextUiSettingsData = getRichTextUiSettingsData(slide);
+	return !defaultBadges.includes(richTextUiSettingsData.achievementKey);
```

Comment:
Make sure to update the Confluence with how to create the different slide types.

Reply:
Yeah, as we have this complete, I'll update that doc, thanks checking!

Reply:
Ticket added!